README.md

```diff
@@ -2,7 +2,58 @@
 
 Kubernetes configuration yaml files used in FFC clusters.
 
-## Priority Classes
+## Helm Charts
+
+### ffc-workstream
+Helm chart for configuring a new workstream namespace
+
+`./helm/ffc-workstream`
+
+Includes:
+- Resource Quota
+- Role Binding
+
+#### Usage
+This Helm chart uses the [FFC Helm Library](https://github.com/DEFRA/ffc-helm-library).
+
+The chart's `values.yaml` contains example values that must be overriden with during helm install/upgrade commands to successfully create namespace resources.
+
+Example values are provided below using the ffc-demo namespace.
+```
+name: ffc-demo
+namespace: ffc-demo
+workstream: ffc-demo
+environment: production
+
+labels:
+  instance:
+  version: 1.0.0
+  component: ffc-demo
```

Comment:
Should be `service` or just removed if you use the library chart labels template (see comments below)